diff --git a/setup.py b/setup.py
index a2bda62..1825277 100644
--- a/setup.py
+++ b/setup.py
@@ -1043,9 +1043,9 @@ class my_build_ext(build_ext):
             raise RuntimeError("No MFC files found!")
 
         for mfc_content in mfc_contents:
-            shutil.copyfile(mfc_content,
-                            os.path.join(target_dir, os.path.split(mfc_content)[1]),
-                            )
+            dest = os.path.join(target_dir, os.path.split(mfc_content)[1])
+            print("Copying '{}' to '{}'".format(mfc_content, dest))
+            shutil.copyfile(mfc_content, dest)
 
 
     def build_exefile(self, ext):
