diff --git a/setup.py b/setup.py
index 736d40f..58dcb0c 100644
--- a/setup.py
+++ b/setup.py
@@ -625,11 +625,10 @@ class my_build_ext(build_ext):
 
     def _why_cant_build_extension(self, ext):
         # Return None, or a reason it can't be built.
-        # Exclude exchange 32-bit utility libraries from 64-bit
-        # builds. Note that the exchange module now builds, but only
-        # includes interfaces for 64-bit builds.
-        if self.plat_name == "win-amd64" and ext.name == "exchdapi":
-            return "No 64-bit library for utility functions available."
+        if ext.name in {'exchdapi', 'exchange'}:
+            # Dont need these modules, and they require headers not present in
+            # the windows SDK
+            return ext.name + " module is disabled"
         if get_build_version() >= 14:
             if ext.name == "exchange":
                 ext.libraries.append("legacy_stdio_definitions")
